bxml/bxmlConcepts.md

```diff
@@ -0,0 +1,64 @@
+# BXML Concepts and Vocabulary
```

Comment:
This looks like it is duplicated from [PR 90](https://github.com/Bandwidth/ap-docs/pull/90). Either close 90 and let's track it here, or remove it from here. (if you want to track here, be sure to add to summary.md)

Reply:
I want it deleted from here

Reply:
https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-git-repo